src/test/java/core/mate/academy/service/MachineServiceTest.java

```diff
@@ -14,7 +14,7 @@
 import java.util.Random;
 
 public class MachineServiceTest {
-    private static MachineService<Machine> machineService;
+    private static MachineService machineService;
```

Comment:
Tests must not be modified, if your test didn't work, that's mean you need to change code, not tests, usually :D

Reply:
By the way, it should work even with that generic, if I'm not mistaken